### DISCUSSION TITLE
Read only column [Jul 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3013] [Discord Threads]
> 🙋‍♂️ **Elf**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@160755405033701377> Not possible at the moment.  
#  
> 🙋‍♂️ **Elf**  
#### OK, thank you.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1001641560594718721)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by bfontaine** [comment]
And is it planned?

**Reply by da-da-da** [reply]
I change col editable property to let some column read only temporarily. But there are some problem in some views.
